src/components/web3-provider.js

```diff
@@ -441,6 +443,17 @@ class Web3Provider extends Component {
     const isProduction = process.env.NODE_ENV === 'production'
     const networkNotSupported = supportedNetworkId != networkId
 
+    // Redirect if we know a DApp instalation that supports their network.
+    if (currentProvider && networkId && isProduction && networkNotSupported) {
+      const url = new URL(window.location)
+      if (networkId==1) {
```

Comment:
I've pushed for use of strict equality in the past. Unless those env vars come out as strings, I would favor using it here.